components/Button/TabButton.tsx

```diff
@@ -19,29 +21,33 @@ export type TabButtonProps = {
 };
 
 const TabButton: React.FC<TabButtonProps> = React.memo(
-	({ title, detail, badge, icon, titleIcon, ...props }) => (
-		<StyledButton noOutline {...props}>
+	({ title, detail, badge, icon, titleIcon, vertical, nofill, ...props }) => (
+		<StyledButton $vertical={vertical} $nofill={nofill} noOutline {...props}>
 			{!!icon && <div>{icon}</div>}
 			<div>
 				<div className="title-container">
 					{titleIcon}
-					<p className="title">{title}</p>
+					<Body className="title">{title}</Body>
 					{!!badge && <div className="badge">{badge}</div>}
 				</div>
 
-				{detail && <p className="detail">{detail}</p>}
+				{detail && (
+					<Body className="detail" mono weight="bold">
```

Comment:
Maybe for a separate PR but would be good to list all these mono text values into a `NumberText` or something as already been some discussion about potentially replacing mono so would make these sort of changes really simple.

Reply:
Yeah. I have notes to make the NumericValue component handle all these cases, but it doesn't currently.